src/preload.ts

```diff
@@ -126,28 +127,34 @@ ipcRenderer.on('injectClientCSS', (_event, _userPrefs: UserPrefs, version: strin
 		const splashCSS = readFileSync(pathJoin($assets, 'splashCss.css'), { encoding: 'utf-8' });
 		injectSettingsCSS(splashCSS, splashId);
 
-		const instructionHider = document.getElementById('instructionHider');
-		if (instructionHider === null) throw "Krunker didn't create #instructionHider";
+		const splashMountElementID = 'uiBase';
+		const uiBaseElement = document.getElementById(splashMountElementID);
+		if (uiBaseElement === null) throw `Krunker didn't create #${splashMountElementID}`;
 
+		const splashBackground = createElement('div', {class: ['crankshaft-loading-background']});
+		if (immersiveSplash) splashBackground.classList.add('immersive');
 		const logoSVG = createElement('svg', {
 			id: 'crankshaft-logo-holder',
 			innerHTML: readFileSync(pathJoin($assets, 'full_logo.svg'), { encoding: 'utf-8' })
 		});
 
 		const clearSplash = (_observer: MutationObserver) => {
 			try {
-				logoSVG.remove();
+				splashBackground.remove();
 				_observer.disconnect();
 			} catch (e) {
 				console.log('splash screen was already cleared.');
 			}
 		};
 
-		instructionHider.appendChild(logoSVG);
+		uiBaseElement.appendChild(splashBackground);
 
 		// i am not sure if you should be injecting more elements into a svg element, but it seems to work. feel free to pr a better version tho.
-		logoSVG.appendChild(createElement('div', { class: 'crankshaft-holder-l', id: '#loadInfoLHolder', text: `v${version}` }));
-		logoSVG.appendChild(createElement('div', { class: 'crankshaft-holder-r', id: '#loadInfoRHolder', text: 'by KraXen72 and contributors' }));
+		logoSVG.appendChild(createElement('div', { class: 'crankshaft-holder-l', id: 'loadInfoLHolder', text: `v${version}` }));
+		logoSVG.appendChild(createElement('div', { class: 'crankshaft-holder-r', id: 'loadInfoRHolder', text: 'Client by KraXen72' }));

```

Comment:
you think this'd look fine even with `KraXen72 and contributors` ? there have been many awesome people contributing, so i wanted to include it

Reply:
I don't think it would look terrible, but ultimately it's up to you. Maybe there could be a randomized contributor name there? `Client by CONTRUBUTOR`?

Reply:
... maybe `[Feature] by CONTRIBUTOR`?